qdrant-landing/content/documentation/tutorials/neural-search.md

```diff
@@ -0,0 +1,347 @@
+---
+title: Create a Simple Neural Search Service
+weight: 14
+---
+
+# Create a Simple Neural Search Service
+
+| Time: 30 min | Level: Beginner | Output: [GitHub](https://github.com/qdrant/qdrant_demo/blob/master/qdrant_demo/init_vector_search_index.py) | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1kPktoudAP8Tu8n8l-iVMOQhVmHkWV_L9?usp=sharing)   |
+| --- | ----------- | ----------- |----------- |
+
+This tutorial shows you how to build and deploy your own neural search service. The created service will be searching through descriptions of companies from [startups-list.com](https://www.startups-list.com/). The website contains the company names, descriptions, their locations and a picture for each entry. 
+
+To create a neural search service, you will need to process your raw data and then create a search function to manipulate it. First, you will download and prepare a sample dataset using a modified version of the BERT ML model. Then, you will load the data into Qdrant, create a neural search API and serve it using FastAPI. 
+
+## Prerequisites
+
+To complete this tutorial, you will need:
+
+- Docker - The easiest way to use Qdrant is to run a pre-built Docker image.  
+- [Raw parsed data](https://storage.googleapis.com/generall-shared-data/startups_demo.json) from startups-list.com.
+- Python version 3.8
+
+> **Note**: The code for this tutorial can be found here: | [Step 1: Data Preparation Process](https://colab.research.google.com/drive/1kPktoudAP8Tu8n8l-iVMOQhVmHkWV_L9?usp=sharing) | [Step 2: Full Code for Neural Search](https://github.com/qdrant/qdrant_demo/blob/master/qdrant_demo/init_vector_search_index.py). |
+
+## Prepare sample dataset 
+
+To conduct a neural search on startup descriptions, you must first encode the description data into vectors. To process text, you need to use a pre-trained language model - DistilBert. The [sentence-transformers](https://github.com/UKPLab/sentence-transformers) library lets you conveniently download and use many pre-trained models, such as DistilBERT.
+
+1. First you need to download the dataset.
+
+```
+wget https://storage.googleapis.com/generall-shared-data/startups_demo.json
+```
+
+2. Use the SentenceTransformer pre-trained model to convert the text into vectors. 
+
+```python
+pip install sentence-transformers
+```
+
+3. Import all relevant models.
+
+```python
+from sentence_transformers import SentenceTransformer
+import numpy as np
+import json
+import pandas as pd
+from tqdm.notebook import tqdm
+```
+
+You will use a Machine Learning model called `distilbert-base-nli-stsb-mean-tokens`.
+DistilBERT is a lightweight version of BERT, which speeds up your service and reduces resource loads.
+
+4. Download and create a pre-trained sentence encoder.
+
+```python
+model = SentenceTransformer('distilbert-base-nli-stsb-mean-tokens', device="cuda")
```

Comment:
Might make sense to switch to the more recent model, e.g. `all-MiniLM-L6-v2`

The only change is the name here